include/lfp/protocol.hpp

```diff
@@ -164,6 +168,14 @@ class unique_lfp {
         this->fp.reset(nullptr);
     }
 
+    /** Implicit conversion to `bool`
+     *
+     *  Checks whether an object is owned.
+     */
+    explicit operator bool() noexcept (true) {
```

Comment:
Question: why if I declare this method "const" (as in unique_ptr), it suddenly stops being picked up for conversion?

Reply:
I suppose the non-const pointer conversion is preferred, since the unique_lfp reference is non-const.